content/getting-started/events-api-best-practices/best-practices.md

```diff
@@ -0,0 +1,88 @@
+# Best practices to use the Events API with your connector
+
+<!--
+- Explication rapide de l'events api + lien vers la référence doc de l'events api + helpcenter
+- Explication rapide d'un use-case d'une extension connecté à l'events api
+  - aka, mon site ecommerce aimerait se sync avec le pim rapidement aprés des modifications sur un produit
+  -->
+
+This page presents best practices and recommendations you might need while implementing the Events API inside an API based connector.
+
+First, if you need help to setup a connector to use the Events API take a loot at the [Quick start my first webhook](/getting-started/quick-start-my-first-webhook/welcome.html) guide.
+
+## Configure your server
+
+TODO
+
+<!--
+- recommandations devops (voir avec Pierre-yves)
+  - DNS, Firewall, TLS certificate (+lib TLS)
+  - TLS car la data doit être sécurisé entre le PIM et l'extension
+  - https://api.akeneo.com/events-documentation/limits-and-scalability.html#extension-side-limits
+-->
+
+## Handle API events asynchronously
+
+### Connector scalability
+
+The Events API can send a lot of requests to your connector endpoint. The default value being 4000 request per hour and each request can contains to up to 10 events (see [limits and scalability](/events-documentation/limits-and-scalability.html#limit-of-event-api-requests-per-hour)).
+
+This means that your connector needs to be able to handle up-to 40,000 events per hour.
+
+But then, each of this event (like `product.updated`) is not necessarily useful on its own, as you might want to gather additional data from the PIM Rest API to be able to process it properly.
+
+For example, you will want to get the attributes and their types for the family of the product, or you will want to fetch more data for the reference entities and assets attributes.
```

Comment:
Not sure of "will want". does it respect your idea to simply write "want" ?

Reply:
I guess that it's more "may want" instead of "will want". Or just keep "want" alone yes, I don't know what is best, what do you think ?

Reply:
I think less words is always a good choice ;)

Reply:
I vote for "may want"

Reply:
```suggestion
For example, you may want to get the attributes and their types for the family of the product, or you may want to fetch more data for the reference entities and assets attributes.
```